packages/documentation-site/patternfly-docs/content/layouts/compass/dashboard.tsx

```diff
@@ -0,0 +1,275 @@
+            </Tooltip>
+          </ActionListItem>
+          <ActionListItem>
+            <Tooltip content="Add">
+              <Button variant="plain" icon={<OutlinedPlusSquare />} aria-label="Add" isCircle />
+            </Tooltip>
+          </ActionListItem>
+        </ActionListGroup>
+        <ActionListItem>
+          <Tooltip content="Assistant">
+            <Button variant="plain" icon={<Icon style={{ translate: ".05em .1em; scale: 1.1" }}><RHAiExperienceIcon /></Icon>} aria-label="Assistant" isCircle className="pf-v6-m-ai-indicator" />
+          </Tooltip>
+        </ActionListItem>
+        <ActionListGroup>
+          <ActionListItem>
+            <Tooltip content="Help">
+              <Button variant="plain" icon={<OutlinedQuestionCircleIcon />} aria-label="Help" isCircle />
+            </Tooltip>
+          </ActionListItem>
+          <ActionListItem>
+            <Tooltip content="Copy">
+              <Button variant="plain" icon={<OutlinedCopy />} aria-label="Copy" isCircle />
+            </Tooltip>
+          </ActionListItem>
+        </ActionListGroup>
+      </ActionList>
+    </CompassPanel>
+  );
+
+  const userDropdownItems = (
+    <>
+      <DropdownItem key="group 2 profile">My profile</DropdownItem>
+      <DropdownItem key="group 2 user">User management</DropdownItem>
+      <DropdownItem key="group 2 logout">Logout</DropdownItem>
+    </>
+  );
+
+  const onDropdownSelect = () => {
+    setIsDropdownOpen(false);
+  };
+
+  const onDropdownToggle = () => {
+    setIsDropdownOpen(!isDropdownOpen);
+  };
+
+
+  const userDropdown = (
+    <Dropdown
+      isOpen={isDropdownOpen}
+      onSelect={() => onDropdownSelect}
+      onOpenChange={(isOpen: boolean) => setIsDropdownOpen(isOpen)}
+      popperProps={{ position: "right" }}
+      toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+        <MenuToggle
+          ref={toggleRef}
+          onClick={onDropdownToggle}
+          isExpanded={isDropdownOpen}
+          variant="plain"
+        >
+          <Flex
+            alignItems={{ default: "alignItemsCenter" }}
+            gap={{ default: "gapMd" }}
+          >
+            Aliyah Frazier
+            <Avatar src={imgAvatar} alt="" size="md" />
+          </Flex>
+        </MenuToggle>
+      )}
+    >
+      <DropdownList>{userDropdownItems}</DropdownList>
+    </Dropdown>
+  );
+
+  const northContent = <CompassHeader logo={<RHAutomationsLogo />} nav={navContent} profile={userDropdown} />;
+
+  const mainContent = (
+    <>
+      <CompassHero gradientDark={{ stop1: '#3d2785', stop2: '#1b0d33', stop3: '#000' }} backgroundSrcDark={heroBg}>
+        <Content>
+          <h1>Automation that does more</h1>
+          <p>
+            Red Hat Ansible Automation Platform offers more capabilities,
+            accessibility, and flexibility, so you can bring the power of
+            automation to the teams, tasks, and environments that need it.
+          </p>
+          <ActionList>
```

Comment:
Can you wrap these `ActionListItem`s in an `ActionListGroup`? The designs have these a little tighter spaced 

<img width="379" height="339" alt="Image" src="https://github.com/user-attachments/assets/7916583b-2229-4a67-89b8-c0adbc19feb8" />